build.gradle

```diff
@@ -35,6 +35,8 @@ repositories {
 dependencies {
     compile gradleApi()
     runtime "io.spring.gradle:dependency-management-plugin"
+    compile "org.jacoco:org.jacoco.core:${jacocoVersion}"
```

Comment:
Let's use the next structure for dependencies

compile ...

runtime ...

testCompile ...

testRuntime ...

without mixes like
compile ...
runtime ...
compile ...